source/common/quic/quic_filter_manager_connection_impl.cc

```diff
@@ -61,10 +62,15 @@ bool QuicFilterManagerConnectionImpl::aboveHighWatermark() const {
 }
 
 void QuicFilterManagerConnectionImpl::close(Network::ConnectionCloseType type) {
-  if (quic_connection_ == nullptr) {
+  if (quicConnection() == nullptr) {
     // Already detached from quic connection.
     return;
   }
+  if (!initialized_) {
```

Comment:
Can we get here?  I thought quicConnection would return null above if not initialized.